src/View/ViewableData.php

```diff
@@ -754,11 +669,9 @@ public function CSSClasses($stopAtClass = ViewableData::class)
 
     /**
      * Return debug information about this object that can be rendered into a template
-     *
-     * @return ViewableData_Debugger
      */
-    public function Debug()
+    public function Debug(): string
     {
-        return new ViewableData_Debugger($this);
+        return ViewableData_Debugger::create($this);
     }
```

Comment:
PHP will automatically call the `__toString()` method to coerce to the correct return type.

Reply:
nvm for the current casting to work it needs to allow returning `ViewableData`.